doc/devel/testing.rst

```diff
@@ -8,6 +8,7 @@ Matplotlib's testing infrastructure depends on pytest_. The tests are in
 :file:`lib/matplotlib/tests`, and customizations to the pytest testing
 infrastructure are in :mod:`matplotlib.testing`.
 
+.. _dependencies: http://matplotlib.org/users/installing.html#dependencies
```

Comment:
This should be a Sphinx cross-references and not http://matplotlib.org/...

Reply:
how do I do that to a particular anchor in the page, i.e. dependencies on the installing page?

Reply:
You'd need to add a tag before the heading, but fortunately, there's already one there: `install_requirements`. You can use that name in the cross-reference instead of `dependencies`.

Reply:
So this change is not needed any more?